riotwatcher/_apis/BaseApi.py

```diff
@@ -46,3 +47,59 @@ def request(self, endpoint_name, method_name, region, url_ext, **kwargs):
                     response = mod
 
         return response
+
+    def request_static(self, version, locale, url_ext):
```

Comment:
The reason why is because the URLs for the version and for the actual file a a bit different. The version take only the region as a path and the static has version, locale and finally the url_extension. I will have to call both request_version and request_static from DataDragonApi, I'll be happy to refactor it, I just assumed it would be cleaner.